apps/sessions-demo/src/components/Home/use-trade.ts

```diff
@@ -39,10 +39,11 @@ export const useTrade = (
         new AnchorProvider(connection, {} as Wallet, {}),
       ).methods
         .exampleTransfer(new BN(100))
-        .accounts({
+        .accountsPartial({
           sessionKey: session.sessionPublicKey,
           sink: sinkAta,
           userTokenAccount: userTokenAccount,
+          mint: NATIVE_MINT,
```

Comment:
Don't merge this yet, I just updated https://github.com/fogo-foundation/fogo-sessions/pull/29 so that `mint` is now an argument of this hook so you'll want to rebase that when I merge once CI passes